### PR TITLE
[FIG] add `debug` subcommand

### PR DESCRIPTION
**Pull request by mschrage**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Add new completions to Fig CLI
**What is the current behavior? (You can also link to an open issue here)**
No support for `debug` subcommand

### CODE REVIEW

File: dev/fig.ts [hunk 1]
```diff
@@ -219,6 +219,61 @@ const completionSpec: Fig.Spec = {
...
+            "Print out terminal configuration and environment variables",
+        },
+        {
+          name: "app",
+          description: "Run app executable directly in terminal",
+        },
+        {
+          name: "sample",
+          description:
+            "Profile a the Fig process to determine why it is hanging or locked",
```
Comment (QuiiBz) [inline dev/fig.ts:240]: Looks like a typo in the description

File: dev/fig.ts [hunk 2]
```diff
@@ -219,6 +219,61 @@ const completionSpec: Fig.Spec = {
...
+                return curr.length == 0 && prev.length > 0;
+              },
+              postProcess: (out, tokens) => {
+                const pivot = tokens.indexOf("logs");
+                const insertedLogFiles = tokens.slice(pivot);
+                return out
+                  .split("\n")
+                  .map((log) => {
+                    return { name: log.replace(".log", "") };
+                  })
```
Comment (QuiiBz) [inline dev/fig.ts:267]: We can simplify this a bit:
```suggestion
                  .map((log) => ({
                    name: log.replace(".log", ""),
                  })
```

File: dev/fig.ts (hunk 1 above)
Comment (mschrage) [inline dev/fig.ts:240]:
```suggestion
            "Profile the Fig process to determine why it is hanging or locked",
```

**Comment by mschrage** [comment]
Not in `1.0.47`. Goes live in `1.0.48`!

**Comment by fedeci** [comment]
Oh, I should join the beta maybe.

**Review by fedeci — APPROVED** [review]

**Review by QuiiBz — APPROVED** [review]
LGTM! Feel free to merge when `1.0.48` is out

**Pull request merged**